modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/AgentsHandler.java

```diff
@@ -81,7 +89,7 @@ private void validateAgentHandshake(AgentHandshakeRequest req) {
         Collection<Account> accounts = accRepo.getAllByTokens(tokens);
 
         if (accounts.isEmpty())
-            throw new IllegalArgumentException("Failed to authenticate with token(s): " + tokens);
+            throw new IllegalArgumentException(messages.getMessage(ERR_FAILED_AUTH_WITH_TOKENS, tokens.toArray()));
```

Comment:
May be we can have method that also accepts collections?

Reply:
Fix in WebConsoleMessageSourceAccessor.